easybuild/tools/package/utilities.py

```diff
@@ -106,9 +106,12 @@ def package_with_fpm(easyblock):
                pprint.pformat([easyblock.toolchain.as_dict()] + easyblock.cfg.dependencies()))
     depstring = ''
     for dep in deps:
-        _log.debug("The dep added looks like %s ", dep)
-        dep_pkgname = package_naming_scheme.name(dep)
-        depstring += " --depends %s" % quote_str(dep_pkgname)
+        if dep.get('external_module', False):
```

Comment:
No, see:

```
>> help({}.get)
Help on built-in function get:

get(...)
    D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None.
```

```
>>> {}.get('external_module', False)
False
>>> {}.get('external_module', default=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: get() takes no keyword arguments
```

I'm using this instead of `dep['external_module']` because the `external_module` key is not defined for the toolchain (yet). And it's safer, anyway.